src/DiskIO/DiskThreads/aiops.cc

```diff
@@ -578,16 +579,16 @@ squidaio_cleanup_request(squidaio_request_t * requestp)
     case _AIO_OP_OPEN:
         if (cancelled && requestp->ret >= 0)
             /* The open() was cancelled but completed */
-            close(requestp->ret);
+            xclose(requestp->ret);
 
         squidaio_xstrfree(requestp->path);
 
         break;
 
     case _AIO_OP_CLOSE:
         if (cancelled && requestp->ret < 0)
-            /* The close() was cancelled and never got executed */
-            close(requestp->fd);
+            /* The xclose() was cancelled and never got executed */
```

Comment:
I would keep the original comment, especially since this PR does not adjust a very similar _AIO_OP_OPEN comment. Ideally, the comment text should not refer to a system call, but we do not have to improve that text in this PR.

Same for the other occurrence, of course.